xbmc/filesystem/IFileTypes.h

```diff
@@ -40,6 +40,13 @@ namespace XFILE
 /* indicate that caller want to reopen a file if its already open  */
   static const unsigned int READ_REOPEN = 0x100;
 
+// file cache buffer modes
+  static constexpr int CACHE_BUFFER_MODE_INTERNET = 0;
```

Comment:
Unsigned int?

Reply:
Not seems necessary because is not used as bitmask, only is compared with settings value which is also `int`:

`settings->GetInt(CSettings::SETTING_FILECACHE_BUFFERMODE) == CACHE_BUFFER_MODE_NETWORK`